olympics/olympics.ipynb

```diff
@@ -0,0 +1,191 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "bdc3cdec",
+   "metadata": {},
+   "source": [
+    "# Template"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3bde5ed6",
+   "metadata": {},
+   "source": [
+    "This is the template project. Its very first content must be a Markdown header (e.g. `# My title`) and should be short and close to the project name."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "884fad20",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.pandas  # noqa\\n\"\n",
+    "import pandas as pd\n",
+    "import panel as pn\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "171d2421",
+   "metadata": {},
+   "source": [
+    "## Data loading\n",
+    "\n",
+    "Loading some data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "89e3186e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = pd.read_csv('data/athlete_events.csv')\n",
```

Comment:
I'd suggest simply just directly reading from URL for those perusing this notebook so they don't have to set up an entire anaconda project lock to use it.
```suggestion
    "df = pd.read_csv('https://media.githubusercontent.com/media/srungta/120-years-of-olympics/master/data/athlete_events.csv')\n",
```

Reply:
Do you mean I should just change the URL to the Kaggle dataset I found it from?

Reply:
> I'd suggest simply just directly reading from URL for those perusing this notebook so they don't have to set up an entire anaconda project env to use it.

Heya, this suggestion makes sense but at the same time there's some tension with how the whole system is supposed to work. Data files are meant to be declared either via the downloads section of `anaconda-project.yml` or via an intake catalog. This allows replacing them before running the tests with smaller datasets (e.g. we copy a small dataset in the local `<project>/data` folder before running the tests). Doing it this way also means the data is either downloaded upfront (download from anaconda-project.yml) or cached (intake) on AE5, every new visitor of the live notebook (and app if there's one) won't have to wait for the data to be downloaded. In this case the CSV is 42MB, not so small but also not so big.

This is a bit described in https://examples.holoviz.org/contributing.html#manage-the-data.

But yeah I also see that it doesn't make it easier for users to run the notebook by just copy/pasting snippet from it. To be fair it's a mode I haven't considered much, many projects require a non-trivial set of dependencies so users would have to go through that first barrier anyway.

So overall I'd prefer if the change was reverted and we'd stick for now to `df = pd.read_csv('data/athlete_events.csv')`.